ogr/services/github/service.py

```diff
@@ -65,7 +65,7 @@ def __init__(
                 total=int(max_retries),
                 read=0,
                 # Retry mechanism active for these HTTP methods:
-                method_whitelist=["DELETE", "GET", "PATCH", "POST", "PUT"],
+                allowed_methods=["DELETE", "GET", "PATCH", "POST", "PUT"],
```

Comment:
Hm, this change seems to cause tests to fail (`E   TypeError: __init__() got an unexpected keyword argument 'allowed_methods'`). Seems like it could be caused by some weird version of urllib in the testing environment. I will drop it for now since the change is not very related to the core of the PR and I just noticed it when running tests locally that they were shouting warnings :)